<launch>
   	
      <node pkg="point_cloud" type="filter_point_cloud" name="filter_point_cloud" output="screen">
         <param name="radius_threshold" value="0.5"/>
         <param name="min_neighbors" value="2"/>
         <param name="voxel_size" value="0.5"/>
         <param name="pass_through_z" value="1"/>
         <param name="pass_through_y" value="3"/>
         <param name="pass_through_x" value="1"/>
         
      </node >     

</launch>

